documentation/docs/reference/dashboards/dashboard-mongodb-replset-summary.md

```diff
@@ -2,26 +2,86 @@
+### Top Hottest Collections by Write
+Shows the five MongoDB collections with the highest write operations (inserts, updates, and deletes) per second. Use this to identify your most frequently modified collections and optimize their write performance.
 
-## Max Heartbeat Time
+### Query Efficiency
+Shows the ratio of documents or index entries scanned versus documents returned. A ratio of 1 indicates optimal query performance where each scanned document matches the query criteria. 
 
-Time span between now and last heartbeat from replicaset members.
+Higher values suggest less efficient queries that scan many documents to find matches. Use this to identify queries that might need index optimization.
 
-## Elections
+### Queued Operations
+Shows the number of operations waiting because the database is busy with other operations. Use this to identify when MongoDB operations are being delayed due to resource conflicts.
 
-Count of elections. Usually zero; 1 count by each healthy node will appear in each election. Happens when the primary role changes due to either normal maintenance or trouble events.
+### Reads & Writes
+Shows both active and queued read/write operations in your MongoDB deployment. Use this to monitor database activity and identify when operations are being delayed due to high load.
 
-## Oplog Recovery Window - by service name
+### Connections
+Shows the number of current and available MongoDB connections. Use this to monitor connection usage and ensure your deployment has sufficient capacity for new client connections.
 
-Timespan 'window' between newest and the oldest op in the Oplog collection.
+### Query Execution Times
+Shows the average latency in microseconds (µs) for read, write, and command operations. Use this metric to monitor query performance and identify slow operations that may need optimization.
+
+## Collection Details
+
+### Size of Collections
+Shows storage size of MongoDB collections across different databases. Use this to monitor database growth and plan storage capacity needs.
+
+### Number of Collections
+Shows the total number of collections in each MongoDB database. Use this to track database organization and growth patterns.
+
+## Replication
+
+### Replication Lag
+Shows how many seconds Secondary nodes are behind the Primary in replicating data. Higher values indicate potential issues with network latency or system resources. The red threshold line at 10 seconds helps identify when lag requires attention.
+
+### Oplog Recovery Window 
+Shows the time range (in seconds) between the newest and oldest operations in the oplog. Use this to ensure sufficient history is maintained for recovery and secondary synchronization.
+
+### Oplog GB/Hour 
+Shows hourly data volume written to cache from the MongoDB oplog by the Primary server. Use this to track oplog growth, plan storage needs, and detect high-write periods. Values are displayed in bytes with hourly intervals.
```

Comment:
Shows the size of the MongoDB oplog generated by the Primary server. Use this to track oplog growth, plan storage needs, and detect high-write periods. Values are displayed in bytes with hourly intervals.